plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/authorization/models/Capabilities.java

```diff
@@ -32,9 +32,19 @@
     @SerializedName("can_search")
     private final boolean canSearch;
 
-    public Capabilities(SupportedAuthType supportedAuthType, boolean canSearch) {
+    @Expose
+    @SerializedName("can_verify_connection")
```

Comment:
@maheshp - As discussed with you, as of now I think the only plugin which would answer to this in negative would be password file auth plugin. However in that case too, usually we see people adding a path that doesn't exist or the file permissions for 'go' user is not setup properly. Since the inbuilt one does not have an option for checking if the setup is proper, people often end up saving their configs with a wrong file and get themselves blocked out. So, I think it would be useful to have such a check even for the password plugins. I agree about the changes around `can_authorize` call, that makes sense.